src/libraries/adaptive-curve/ConstantsLib.sol

```diff
@@ -14,12 +19,12 @@ library ConstantsLib {
     /// @notice Target utilization (scaled by WAD).
     int256 public constant TARGET_UTILIZATION = 0.9 ether;
 
-    /// @notice Initial rate at target per second (scaled by WAD).
-    int256 public constant INITIAL_RATE_AT_TARGET = int256(0.04 ether) / 365 days;
+    /// @notice Initial rate at target per second (scaled by WAD) (1% min APR / 16% max APR).
+    int256 public constant INITIAL_RATE_AT_TARGET = 4_00 * ONE_BPS / 365 days;
 
-    /// @notice Mininimum rate at target per second (scaled by WAD) (0.1% APR).
-    int256 public constant MIN_RATE_AT_TARGET = int256(0.001 ether) / 365 days;
+    /// @notice Minimum rate at target per second (scaled by WAD) (0.025% min APR).
+    int256 public constant MIN_RATE_AT_TARGET = 10 * ONE_BPS / 365 days;
 
-    /// @notice Maximum rate at target per second (scaled by WAD) (1B% APR).
-    int256 public constant MAX_RATE_AT_TARGET = int256(0.01e9 ether) / 365 days;
+    /// @notice Maximum rate at target per second (scaled by WAD) (800% max APR).
```

Comment:
Max rate at target is 200% APR (max rate is 800% APR after applying curve steepness). Right now it reads like max rate at target is 800% APR

Reply:
Yes, hence my initial suggestion:
```suggestion
    /// @notice Maximum rate at target per second (scaled by WAD) (200% APR => 800% max APR).
```

But it got rejected by @MathisGD and I agreed because a rate at target cannot be named "APR" because it only makes sense in the context of the IRM IMO

We're circling back here and I get bored because it's the fifth change and none is perfect
Waiting on others thoughts

Reply:
I propose this
```suggestion
    /// @notice Maximum rate at target per second (scaled by WAD).
    /// @dev 200% maximum rate at target leads to a 800% max rate.
```